### ISSUE TITLE
Integration tests with pytest for docker and docker-compose  

### ISSUE THREAD

**Issue by finleymercyabraham** [amosproj/amos-ss2021-synthetic-file-system#31]
## User story
1. As a software developer
2. I want the test case that tests the functionality of FUSE inside the docker.
3. So that errors would be detected early

## Acceptance criteria
* Tests cases that match the use case.
* Integration which tests where the Docker container has started
* Tests whether traversing through the Metadata is  working 

## Definition of done

      1. Code review from peers has been completed and passed
        a. Code is fully implemented and commented
        b. Adheres to coding guidelines
        c. Commits are not squashed
        d. Code is checked into the repository
    2. Unit tests/ Integration test have been written and are passing
    3. Cleanliness of the Code
    4. No critical bugs are open
    5. Documentation has been updated
    6. Code has been included into the release (candidate)
    7. Code Coverage:
         60 % - for Features
         70 % - for Sprint Release
         80 % - for Product Release

**Comment by vaidehi-for-coding** [comment]
@marlonweghorn I have pushed my changes to a new branch sfs_integ_test. 
I am executing the command pytest-3 -c setup.cfg. This executes all tests in unittest files. The file I'm interested in running is test_docker.py. It executes commands from docker/Dockerfile one by one. It gets stuck at COPY statements, because it cannot find any of the files it is supposed to copy from the repo to the container. It could be a problem with the context field in the docker-compose.yml, but if it runs manually, it should also run via a pytest. I tried modifying it to ADD, instead of COPY, and played around with suggested google solutions, doesn't work.
